ci/continuous-builder.sh

```diff
@@ -114,6 +127,7 @@ while true; do
     git credential-store --file ~/.git-creds store
     git config --global credential.helper "store --file ~/.git-creds"
 
+    set-github-status -c ${STATUS_REF} -s $CHECK_NAME/pending || true
```

Comment:
Are you sure about this? We do not want to reset the status in case a PR is already green, no?

Reply:
If I am not mistaken, `list-branch-pr` excludes PRs with a green state? I did not check it. In any case I have accidentally added this line to the PR, let me remove it for now.

Reply:
I have removed the line in the new iteration.